src/main/java/no/nav/familie/integrasjoner/dokarkiv/DokarkivService.kt

```diff
@@ -46,17 +46,23 @@ class DokarkivService(private val dokarkivClient: DokarkivClient,
 
         val metadata = dokarkivMetadata.getMetadata(arkiverDokumentRequest.dokumenter[0])
         val arkivdokumenter = arkiverDokumentRequest.dokumenter.map(this::mapTilArkivdokument)
+        val jpsak: Sak?= if(arkiverDokumentRequest.fagsakId!= null) Sak(
```

Comment:
Hvorfor bør fagsakId kunne være null? Kan vi ikke gjøre dette til et required felt i body'en?

Reply:
Vi har ikke fagsakId idet vi arkiverer søknader f.eks

Reply:
> > @tnarland @MengZhuNAV, have you talked to Birgitte about these fields? I would recommend them getting the correct information so that we dont use time on guessing brevkode and fagsaksystem etc.
> > In addition I think many of these fields should be defined by the client and not integrasjoner. Remember that this app is a proxy for BA, EF and KS.
> 
> The thing is they don't exist yet, but it shouldn't stop us from getting it up and running, which has been our main focus. I agree with the last part btw.

Ok, that is fine and great! I suggest moving the "hardcoded" (not figured out fields) to ba-sak and moving towards the goal. Just remember to create a card or comment the task in Favro 👍

Reply:
The hardcoded fields follow the existing architecture which is also used by kontanstøtte project. We hardcoded metadata not only for the barnetrygd document, but also other documents. I have not done a thorough evaluation of the cost and profit of changing the architecture, but I think that should be another PR anyway.